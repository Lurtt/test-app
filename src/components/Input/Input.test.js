import React from 'react'
import renderer from 'react-test-renderer'

import Input from './'

describe('<Input />', () => {
  test('Should render', () => {
    const result = renderer.create(<Input />).toJSON()
    expect(result).toMatchSnapshot()
  })

  test('Should accept {...props}', () => {
    const result = renderer.create(<Input id={1} value="some text" />).toJSON()
    expect(result).toMatchSnapshot()
  })
})

// test('Link changes the class when hovered', () => {
//   const component = renderer.create(
//     <Link page="http://www.facebook.com">Facebook</Link>,
//   );
//   let tree = component.toJSON();
//   expect(tree).toMatchSnapshot();
//
//   // manually trigger the callback
//   tree.props.onMouseEnter();
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();
//
//   // manually trigger the callback
//   tree.props.onMouseLeave();
//   // re-rendering
//   tree = component.toJSON();
//   expect(tree).toMatchSnapshot();
// });
