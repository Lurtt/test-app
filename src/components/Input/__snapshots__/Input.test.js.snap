// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> Should accept {...props} 1`] = `
<input
  className="sc-bdVaJa YQfmf"
  id={1}
  type="text"
  value="some text"
/>
`;

exports[`<Input /> Should render 1`] = `
<input
  className="sc-bdVaJa YQfmf"
  type="text"
/>
`;
